training/automl/components/automl_image_classification/spec.yaml

```diff
@@ -1,7 +1,7 @@
 $schema: http://azureml/sdk-2-0/AutoMLComponent.json
 name: microsoft_azureml_automl_image_classification_component
-display_name: AutoML Image Classification (Private Preview)
+display_name: AutoML Image Classification
 description: Component that executes an AutoML Image Classification task model training in a pipeline.
```

Comment:
I would restructure the description --
"Component that kicks off an AutoML job to train an image classification model within an Azure Machine Learning pipeline." 

Same structure for all the descriptions.

Reply:
Updated